### ISSUE TITLE
Run task to ensure access and copy AMI snapshot

### ISSUE THREAD

**Issue by infinitewarp** [cloudigrade/cloudigrade#167]
As a Doppler admin, I want snapshots copied via Celery tasks so that they can be done asynchronously.

For each AMI ID we encounter that needs inspection, we need to do the following actions before we can create a volume (in a later task):

- check the AMI metadata to get the snapshot ID
- take ownership of the snapshot ID
- copy the snapshot to `us-east-1a` (or whatever default region + availability zone we have configured for our cluster)

## Acceptance Criteria 

- [x] Verify that discovering a new AMI via account creation calls this new task
  - [x] Verify that our AWS account has appropriate access to a copy of the AMI snapshot
  - [x] Verify that we have a copy of the snapshot in our region + AZ after this task has completed

## Tech assumptions

- Retry should use an exponential backoff [like this example](https://stackoverflow.com/a/9752811/6621995).
- So far we are assuming that all AMIs we care about are EBS-backed for this year (per Tim). We are NOT touching S3-backed AMIs. We think S3-backed AMIs are uncommon, but we should check on this again later…
- For this process to start, we also need to change the output from https://github.com/cloudigrade/cloudigrade/issues/88 to call this new task for each AMI ID *instead of* enqueuing a message for each one.
- We will have another story later to call this task from the log analyzer.
- This section of the AWS [Regions and Availability Zones](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html) doc reminds us that we MUST copy the snapshot because we cannot trust that we actually have the same AZ as our customer's account:
   > To ensure that resources are distributed across the Availability Zones for a region, we independently map Availability Zones to identifiers for each account. For example, your Availability Zone us-east-1a might not be the same location as us-east-1a for another account. There's no way for you to coordinate Availability Zones between accounts. 
- According to [EBS Copy Snapshot] (https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-copy-snapshot.html)
   > Each account can have up to 5 concurrent snapshot copy requests to a single destination region. 
- When we tell AWS API to copy the snapshot, we expect AWS to give some reasonable error if the 5-concurrent limit has been hit. In this case, we need to trigger a Celery task retry.

**Comment by adberglund** [comment]
https://asciinema.org/a/obJLfd5vtYVjQPH9pxutcFGvN
